private/bufpkg/bufcheck/buflint/buflint_test.go

```diff
@@ -962,6 +1038,7 @@ func testLint(
 		relDirPath,
 		nil,
 		nil,
+		"buf",
```

Comment:
This should be in a deeper subdirectory to make sure it is clear to future developers what this is for - I'd suggest `deps/protovalidate/buf/validate/validate.proto`, with this value being changed to `deps/protovalidate`.

Reply:
resolved in [1b2c71c](https://github.com/bufbuild/buf/pull/2377/commits/1b2c71cceeb54702ea58bc69970a1a434fb6410e)